web/src/pages/PluginTemplate/List.tsx

```diff
@@ -182,13 +182,18 @@ const Page: React.FC = () => {
       />
       <RawDataEditor
         visible={visible}
-        type="service"
-        readonly={true}
+        type="PluginTemplate"
+        readonly={false}
         data={rawData}
         onClose={() => {
           setVisible(false);
         }}
         onSubmit={(data: any) => {
+          handleTableActionSuccessResponse(
+            `${formatMessage({ id: 'component.global.submit' })} ${formatMessage({
+              id: 'component.status.success',
+            })}`,
+          );
```

Comment:
we can use the notification, refer to `notification` https://ant.design/components/notification-cn/